### DISCUSSION TITLE
[eForms-SDK] Name difference for a same field in two different eforms

### DISCUSSION THREAD

**Discussion by rlagrange98** [OP-TED/eForms-SDK#1033] [Q&A] [answered]
Hi,

I saw that for eforms F29 field name was : GR-Unpublish-BT-171 and for eforms F38 it was GR-Unpublish-BT-171-Tender. (noticetype)
In "translation" > "group" its also written as GR-Unpublish-BT-171.

So I was wondering why ?

Best regards,
Romain Lagrange

**Comment by bertrand-lorentz** [comment, marked as answer]
A small correction: "GR-Unpublish-BT-171" is not the identifier of a field, but of a group, which is a container for fields in the notice type definitions.
We do try to have the same identifier for the same "things" across the various notice type definitions, but it seems we missed this one.
It will be corrected in a future version of the SDK.

**Reply by rlagrange98** [reply]
Ok, thank you Bertrand !
